plugins/omemo/po/sv.po

```diff
@@ -37,6 +37,7 @@ msgstr "Hantera enheter"
 msgid ""
 "%s does not trust this device. That means, you might be missing messages."
 msgstr ""
+"%s litar inte på den här enheten. Du kanske inte ser alla meddelanden."
```

Comment:
"You might not see all messages" instead of "missing messages", hope that works. Not sure if I have seen this in the UI myself.